variables.tf

```
@@ -19,6 +19,18 @@ variable "sku" {
   default     = "Basic"
 }
 
+variable "system_assigned_identity_enabled" {
+  description = "Should the system-assigned identity be enabled for this Web App?"
+  type        = bool
+  default     = false
+}
+
+variable "identity_ids" {
+  description = "A list of IDs of managed identities to be assigned to this Web App."
```

Correct resource type:

```suggestion
  description = "A list of IDs of managed identities to be assigned to this Service Bus namespace."
```

---

Good catch 👍